src/pages/home/owners/OwnersTable.tsx

```diff
@@ -59,6 +59,7 @@ export default function OwnersTable({
     offset: (page - 1) * rowsPerPage,
     limit: collaboratorsToRequest,
     phrase: query,
+    excludeMe: true,
```

Comment:
Is this getting added to the backend query parameters? Currently doesn't remove self from collaborators.

Reply:
Yes, this will get added in the BE shortly, i pushed up the change so i wasnt stuck from completing other things, I wont merge it until i test it of course ;)